### DISCUSSION TITLE
Issue trying to set up CI in Azure DevOps

### DISCUSSION THREAD

**Discussion by motoyugota** [OrchardCMS/OrchardCore#16123] [Q&A]
I have an OrchardCore CMS site I am trying to set up in Azure DevOps and I am getting the following error when it is trying to download NuGet packages:

##[error]The nuget command failed with exit code(1) and error(NU1202: Package Shortcodes 1.3.0 is not compatible with netcoreapp3.1 (.NETCoreApp,Version=v3.1). Package Shortcodes 1.3.0 supports: netstandard2.1 (.NETStandard,Version=v2.1)

The site runs fine and downloads the package without issue locally when set up for 3.1 (or even .Net 5), but when Azure DevOps tried to build, it fails.

Can't run under v2.1 because OrchardCore.Application.Cms.Targets won't run in 2.1.

Has anyone gotten this running this way? What do I need to do to make this work?

**Comment by sebastienros** [comment]
> Can't run under v2.1 because OrchardCore.Application.Cms.Targets won't run in 2.1.

OC only support netcoreapp3.1, net5.0 and net6.0, so right you can't use netcoreapp2.1 (note it's not related to netstandard2.1)

> NU1202: Package Shortcodes 1.3.0 is not compatible with netcoreapp3.1 (.NETCoreApp,Version=v3.1). Package Shortcodes 1.3.0 supports: netstandard2.1 (.NETStandard,Version=v2.1)

I believe it's a Nuget.exe issue. Maybe try to get a more recent nuget.exe that the one it's using, or maybe use the one from dotnet sdk.
